packages/storybook/src/stories/ino-chip/ino-chip.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, Locator, test } from '@playwright/test';
+import { goToStory } from '../test-utils';
+
+test.describe('ino-chip', () => {
+  let chip: Locator;
+
+  test.beforeEach(async ({ page }) => {
+    await goToStory(page, ['buttons', 'ino-chip', 'default']);
+    chip = page.locator('ino-chip');
+  });
+
+  test('should render chip', async () => {
+    await expect(chip).toBeInViewport();
+  });
+
+  test('should not render remove icon when removable is not set', async () => {
+    await expect(chip.locator('ino-icon')).not.toBeInViewport();
+  });
+
+  test('should render remove icon when removable is set true', async () => {
+    await chip.evaluate((el) => el.setAttribute('removable', 'true'));
+    await expect(chip.locator('ino-icon')).toBeInViewport();
+  });
+
+  test('should render checkmark if selectable and selected is set', async () => {
+    await chip.evaluate((el) => {
+      el.setAttribute('selectable', 'true');
+      el.setAttribute('selected', 'true');
+    });
+
+    await expect(chip.locator('svg')).toBeInViewport();
```

Comment:
see above

Reply:
also, instead of checking for classes or elements, it's recommended to test for ARIA attributes.
![image](https://github.com/inovex/elements/assets/22963121/b8eae45b-1c37-4be1-bb8e-58b7a4ef6b48)

e.g.
```ts
		const selectedMarker = chip.locator('[aria-selected]')
    // expect(...).toHaveAttribute(...)
```

Reply:
By doing this I found some a11y issues 😄 and created a draft issue.